org.mixedrealitytoolkit.input/Controllers/ArticulatedHandController.cs

```diff
@@ -30,13 +30,6 @@ public class ArticulatedHandController : ActionBasedController
         /// <remarks>Expected to be XRNode.LeftHand or XRNode.RightHand.</remarks>
         public XRNode HandNode => handNode;
 
-        /// <summary>
-        /// Is the hand ready to select? Typically, this
-        /// represents whether the hand is in a pinching pose,
-        /// within the FOV set by the aggregator config.
-        /// </summary>
-        public bool PinchSelectReady => (currentControllerState is ArticulatedHandControllerState handControllerState) && handControllerState.PinchSelectReady;
```

Comment:
Do not remove functionality from our deprecated classes.  Leave as is and mark this class as deprecated.

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/a2d8ed684f96e6231f1bea0a283b8e0ed9b265ba , resolving.